source/package-json.d.ts

```diff
@@ -208,26 +208,12 @@ declare namespace PackageJson {
 	*/
 	export type Dependency = Partial<Record<string, string>>;
 
-	/**
-	Conditions which provide a way to resolve a package entry point based on the environment.
-	*/
-	export type ExportCondition = LiteralUnion<
-	| 'import'
-	| 'require'
-	| 'node'
-	| 'node-addons'
-	| 'deno'
-	| 'browser'
-	| 'electron'
-	| 'react-native'
-	| 'default',
-	string
-	>;
```

Comment:
I can no longer use `ExportCondition` as the key type. However, the only requirement for an export condition is that it should be a string.

Keeping a list of common export conditions will only add to the maintenance workload, and provide little value for autocompletion, so I suggest removing it.